phpunit.evergreen.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.3/phpunit.xsd"
+        backupGlobals="false"
+        backupStaticAttributes="false"
+        colors="true"
+        convertErrorsToExceptions="true"
+        convertNoticesToExceptions="true"
+        convertWarningsToExceptions="true"
+        stopOnFailure="false"
+        syntaxCheck="false"
+        bootstrap="tests/bootstrap.php"
+>
+
+    <php>
+        <ini name="error_reporting" value="-1"/>
```

Comment:
Oh, this may relate to my earlier comment about customizing `php.ini`. Are there other options we might want to include here to ensure things like memory leaks get reported? Alternatively, would it make more sense to just move this into the PR's `php.ini` file that you apply to the entire PHP runtime?

Reply:
This was copied over from phpunit.xml.dist, but I believe we can remove this once we've dealt with php.ini in our toolchain.